presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueries.java

```diff
@@ -4704,11 +4704,9 @@ public void testAccessControl()
         assertAccessAllowed("SHOW STATS FOR lineitem", privilege("orders", SELECT_COLUMN));
         assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)");
         assertAccessAllowed("SHOW STATS FOR (SELECT * FROM lineitem)", privilege("orders", SELECT_COLUMN));
-        assertAccessDenied("SHOW STATS FOR (SELECT * FROM lineitem)",
-                "Cannot select from columns \\[returnflag, linenumber, orderkey, quantity, shipmode, discount, tax, suppkey, partkey, shipinstruct, linestatus, extendedprice, comment, receiptdate, commitdate, shipdate] in table or view .*.lineitem.*",
-                privilege("lineitem", SELECT_COLUMN));
-        assertAccessDenied("SHOW STATS FOR (SELECT nationkey FROM nation)", "Cannot select from columns \\[nationkey\\] in table or view .*.nation.*", privilege("nation", SELECT_COLUMN));
-        assertAccessDenied("SHOW STATS FOR (SELECT * FROM nation)", "Cannot select from columns \\[nationkey, regionkey, name, comment\\] in table or view .*.nation.*", privilege("nation", SELECT_COLUMN));
+        assertAccessDenied("SHOW STATS FOR (SELECT * FROM lineitem)", "Cannot show stats for table .*.lineitem", privilege("lineitem", SELECT_COLUMN));
+        assertAccessDenied("SHOW STATS FOR (SELECT nationkey FROM nation)", "Cannot show stats for table .*.nation", privilege("nation", SELECT_COLUMN));
```

Comment:
When one column is denied access but others are allowed, do we want to indicate this anyhow?

Reply:
It's not an information leak (security related)?

Reply:
It's what we do for regular SELECT queries

Reply:
👍

Reply:
Done